tests/PSRule.Rules.Azure.Tests/Azure.AppGw.Tests.ps1

```diff
@@ -192,8 +192,8 @@ Describe 'Azure.AppGW' -Tag 'Network', 'AppGw' {
             # Pass
             $ruleResult = @($filteredResult | Where-Object { $_.Outcome -eq 'Pass' });
             $ruleResult | Should -Not -BeNullOrEmpty;
-            $ruleResult.Length | Should -Be 5;
-            $ruleResult.TargetName | Should -Be 'appgw-A', 'appgw-B', 'appgw-C', 'appgw-D', 'appgw-G';
+            $ruleResult.Length | Should -Be 3;
+            $ruleResult.TargetName | Should -Be 'appgw-C', 'appgw-D', 'appgw-G';
```

Comment:
Lets do at least one check that the `appgw-A` and `appgw-B` App Gateways are None.
No need to do it in the configuration tests below.
Similar to [this](https://github.com/Azure/PSRule.Rules.Azure/blob/e8510b7fe5971d189ec28f6e34959adec1f9e0b1/tests/PSRule.Rules.Azure.Tests/Azure.AppService.Tests.ps1#L230-L250).

Reply:
Added 🙂 . Thanks for spotting that.